src/couch/src/couch_debug.erl

```diff
@@ -289,12 +292,12 @@ help(print_linked_processes) ->
         initial Pid to start from. The function doesn't recurse to pids
         older than initial one. The output would look like similar to:
         ```
-couch_debug:print_linked_processes(whereis(couch_index_server)).
-name                                         | reductions | message_queue_len |  memory
-couch_index_server[<0.288.0>]                |   478240   |         0         |  109696
-  couch_index:init/1[<0.3520.22>]            |    4899    |         0         |  109456
-    couch_file:init/1[<0.886.22>]            |   11973    |         0         |  67984
-      couch_index:init/1[<0.3520.22>]        |    4899    |         0         |  109456
+        couch_debug:print_linked_processes(whereis(couch_index_server)).
```

Comment:
The reason I indented it on the left is to help visually impaired people like myself to use bigger fonts. It causes additional indentation when you run `couch_debug:help(print_linked_processes)` in the `remsh`. With bigger font the table might not fit into a screen width and all formatting appear broken. It is not the case for this particular table, so I am fine with this change. Just wanted to provide rational for the previous choice of indentation.

Reply:
Ah understood. I indented it to stay consistent.